### PR TITLE
Params for build and test CI scripts on Databricks

### PR DESCRIPTION
**Pull request by NvTimLiu**
Params for build and test CI scripts on Databricks
 
To fix https://github.com/NVIDIA/spark-rapids/issues/9903

Add params to support specify build, test args on Databricks for CI scripts, e.g.

   `python jenkins/databricks/run-build.py -e SKIP_DEP_INSTALL=1` : Run maven build on Databricks and skip installing dependencies into maven repo

   `python jenkins/databricks/run-test.py -e TEST_MODE=DELTA_LAKE_ONLY` : only run Delta Lake  test cases on Databricks

Signed-off-by: timl@nvidia.com

### CODE REVIEW

**Review by jlowe — DISMISSED** [review]

**Review by tgravescs — DISMISSED** [review]

**Comment by razajafri** [comment]
Is there a way to test this?

**Review by pxLi — APPROVED** [review]

File: jenkins/databricks/build.sh [hunk 1]
```diff
@@ -130,6 +130,10 @@ install_dependencies()
 ##########################
 # Main script starts here
 ##########################
+## 'foo=abc,bar=123,...' to 'export foo=abc bar=123 ...'
+if [ -n "$EXTRA_ENVS" ]; then
```
[the remark this answers is not in the document]
Comment (NvTimLiu) [reply jenkins/databricks/build.sh:135]: > do we want to print these out in the above function " echo "Initializing build for Databricks:""

Sounds good to me, updated.

**Comment by pxLi** [comment]
build

**Comment by pxLi** [comment]
re-kicked after merging the fix

**Review by pxLi — APPROVED** [review]

**Review by tgravescs — APPROVED** [review]

**Pull request merged**